samples/sample_multi_transcode/src/transcode_utils.cpp

```diff
@@ -201,7 +201,8 @@ void TranscodingSample::PrintHelp()
         MSDK_STRING("                In encoding sessions (-o::source) and transcoding sessions \n") \
         MSDK_STRING("                  this parameter limits number of frames sent to encoder.\n"));
 
-    msdk_printf(MSDK_STRING("  -ext_allocator    Force usage of external allocators\n"));
+    msdk_printf(MSDK_STRING("  -ext_allocator    Force usage of external allocators (default)\n"));
+    msdk_printf(MSDK_STRING("  -opaq         Force usage of OPAQ memory\n"));
```

Comment:
Why do we have per component sys or vid and global "opaq". I assume we need to align here. Make either all types per component or make them global.

Reply:
this "opaq" key is just ability to return previous behavior, so it is global as well as "sys" and "ext_allocator" keys. I don't know how those per-component types should work with this key (it will work as as it did previously actually) and why did we need them. This option is required because user will not have ability to use opaq after default change

Reply:
So my point is that I don't want to bring redesign of these keys with this current PR, trying to minimize the change. I touched those lines only to group all memory related keys together

Reply:
> So my point is that I don't want to bring redesign of these keys with this current PR, trying to minimize the change. I touched those lines only to group all memory related keys together

I understand that you want to minimize efforts. However do you agree after the change code looks worse?

Reply:
Not sure if it become worse, it become just a little bit different. I agree that logic of setting memtypes seems to be missleading and a little strange. We need all those keys and functionality because it is used in tests, so your suggestion is to rework this part completely, did I get it right?